omf-python/src/attribute.rs

```diff
@@ -218,6 +239,64 @@ impl PyAttributeDataMappedTexture {
     }
 }
 
+#[gen_stub_pyclass]
+#[pyclass(name = "AttributeDataProjectedTexture")]
+/// A texture defined as a rectangle in space projected along its normal.
+///
+/// Behavior of the texture outside the projected rectangle is not defined.
+/// The texture might repeat, clip the element, or itself be clipped to reveal the flat color of the element.
+///
+/// The attribute location must be Projected.
+pub struct PyAttributeDataProjectedTexture(pub AttributeData);
+
+#[gen_stub_pymethods]
+#[pymethods]
+impl PyAttributeDataProjectedTexture {
+    #[getter]
+    /// Array with Image type storing the texture image.
+    fn image(&self) -> PyResult<PyImageArray> {
+        match &self.0 {
+            AttributeData::ProjectedTexture { image, .. } => Ok(PyImageArray(image.clone())),
+            _ => Err(PyValueError::new_err(
+                "AttributeData variant is not supported",
+            )),
+        }
+    }
+
+    #[getter]
+    /// Orientation of the image.
+    fn orient(&self) -> PyResult<PyOrient2> {
+        match &self.0 {
+            AttributeData::ProjectedTexture { orient, .. } => Ok(PyOrient2(*orient)),
```

Comment:
Just a question, why do we need to dereference orient here and width/height below, are there other places we should be using this pattern?

Reply:
Actually it seems I don't need that if I use `match self.0` instead of `match &self.0`. I might change it to that.

Also btw Orient2 has the Copy macro so clippy complains if I use `clone()` there.

Reply:
ah I see, cool this all looks good to me :+1: